code/modules/vehicles/speedbike.dm

```diff
@@ -24,6 +24,491 @@
 	icon_state = "speedbike_red"
 	overlay_state = "cover_red"
 
+
+// ATMOBILE AND ATMOBILE UNIQUE INTERNAL ITEMS
+
+
+
+/obj/vehicle/space/speedbike/atmos
+	name = "prototype atmos vehicle"
+	desc = "This vehicle possesses unparalleled utility for atmospheric containment and control"
+	icon_state = "atmo_bike"
+	overlay_state = "cover_atmo"
+	var/obj/machinery/portable_atmospherics/scrubber/huge/internal_scubber = null
+	var/obj/item/extinguisher/vehicle/internal_extinguisher = null
+	var/ex_out = FALSE
+	var/obj/machinery/portable_atmospherics/canister/proto/default/oxygen/CAN = null
+	var/loaded = TRUE
+	light_range = 7
+
+/obj/vehicle/space/speedbike/atmos/New()
+	. = ..()
+	internal_scubber = new /obj/machinery/portable_atmospherics/scrubber/huge(src)
+	internal_scubber.on = 1
+	internal_extinguisher = new /obj/item/extinguisher/vehicle(src)
+	internal_extinguisher.vehicle = src
+	CAN = new /obj/machinery/portable_atmospherics/canister/oxygen(src)
+//	CAN = new /obj/machinery/portable_atmospherics/canister/proto/default/oxygen(src)
```

Comment:
CAN is /obj/machinery/portable_atmospherics/canister/proto/default/oxygen though?

Reply:
Ah that was a relic from when I first atomized this code and I wasn't sure if prototype canisters would be merged. I had intended (see OP) to revisit it but I didn't realize I left a comment behind.